@navikt/core/css/copybutton.css

```diff
@@ -69,6 +69,8 @@
 
 .navds-copybutton--icon-only {
   --__ac-copybutton-padding: var(--a-spacing-3);
+
+  aspect-ratio: 1;
```

Comment:
🗿 
Easier to just use `min-width` here just in case this actually happens haha.

Seems like the chromatic enable false was just placed on the wrong story, moved it up to the story config from "Default"-story